src/Map/src/UXMapBundle.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\UX\Map;
+
+use Symfony\Component\AssetMapper\AssetMapperInterface;
+use Symfony\Component\Config\Definition\Configurator\DefinitionConfigurator;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
+use Symfony\Component\HttpKernel\Bundle\AbstractBundle;
+use Symfony\UX\Map\Bridge as MapBridge;
+use Symfony\UX\Map\Renderer\AbstractRendererFactory;
+use Symfony\UX\Map\Renderer\NullRendererFactory;
+
+/**
+ * @author Hugo Alliaume <hugo@alliau.me>
+ */
+final class UXMapBundle extends AbstractBundle
+{
+    protected string $extensionAlias = 'ux_map';
+
+    /**
+     * @var array<string, array{ renderer_factory: class-string<AbstractRendererFactory> }>
+     */
+    private static array $bridges = [
+        'google' => ['renderer_factory' => MapBridge\Google\Renderer\GoogleRendererFactory::class],
+        'leaflet' => ['renderer_factory' => MapBridge\Leaflet\Renderer\LeafletRendererFactory::class],
+    ];
+
+    public function configure(DefinitionConfigurator $definition): void
+    {
+        $rootNode = $definition->rootNode();
+        $rootNode
+            ->children()
+                ->scalarNode('renderer')->defaultNull()->end()
+            ->end()
+        ;
+    }
+
+    public function loadExtension(array $config, ContainerConfigurator $container, ContainerBuilder $builder): void
+    {
+        $container->import('../config/services.php');
+
+        if (!isset($config['renderer'])) {
+            $config['renderer'] = 'null://null';
+        }
+
+        if (str_starts_with($config['renderer'], 'null://')) {
+            $container->services()
+                ->set('ux_map.renderer_factory.null', NullRendererFactory::class)
+                ->arg(0, array_map(fn ($name) => 'symfony/ux-map-'.$name, array_keys(self::$bridges)))
+                ->tag('ux_map.renderer_factory');
+        }
+
+        $renderers = ['main' => $config['renderer']];
```

Comment:
"main" ?

Reply:
Renamed to `default`, but it's for the future when we will have multiple renderers per app 🧐